music_assistant/server/providers/tidal/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "music",
+  "domain": "tidal",
+  "name": "Tidal",
+  "description": "Support for the Tidal streaming provider in Music Assistant. Hit the button below to authenticate with Tidal, and check that the redirect is not blocked by a Pop-up Blocker!",
+  "codeowners": ["@jozefKruszynski"],
+  "requirements": ["git+https://github.com/jozefKruszynski/python-tidal.git"],
```

Comment:
is temporary unversioned ?

Reply:
Honestly, I'm not sure what you mean here... Yes is my temporary answer to a "hopefully" temporary issue

Reply:
Yeah, well, by putting only the github version here its no longer pinned to a specific version but guess that is not a big deal for now. You could also pin it so a specific branch, tag or even commit if I'm correct but as you are in control of the repo yourself it shouldn't matter that much.

Reply:
Got ya, I was trying to create a tagged version last night, or perhaps this morning, but I had been out having "fun" for a bit too long to be capable of any code related activities

Reply:
Tagged version now added